src/Worker/ThreadUpdate.php

```diff
@@ -1,11 +0,0 @@
-<?php
-namespace Friendica\Worker;
-
-require_once("include/threads.php");
-
-class ThreadUpdate {
```

Comment:
Isn’t this mentioned anywhere?

Reply:
We don't need that anymore. It was part of old update routines that we removed.